app/src/components/common/icons/index.ts

```diff
@@ -23,3 +23,4 @@ export { IconSearchGrey } from './IconSearchGrey'
 export { IconKleros } from './IconKleros'
 export { IconExclamation } from './IconExclamation'
 export { IconSchedule } from './IconSchedule'
+export { IconSpinner } from './IconSpinner2'
```

Comment:
What do you think about renaming this file from `IconSpinner2` to just `IconSpinner`?

Reply:
well that's where i had problem i don't know why but when i rename it to just IconSpinner it converts to like text file and it's not usable...

Reply:
@hexyls 
![Screenshot 2020-11-24 at 12 27 49](https://user-images.githubusercontent.com/33226956/100088640-ba223f00-2e50-11eb-9c90-1bb4e9fcbccb.png)
here is my concern when i change the name you see how IconSpinner next to it it's like text file and also this error like it's not exported good...

Reply:
Is this just an issue with your IDE identifying the file type? Seems to work fine for me